apps/ledger-live-mobile/src/locales/de/common.json

```diff
@@ -3540,19 +3531,6 @@
       "de" : "Deutsch"
     }
   },
-  "deviceLocalization" : {
```

Comment:
this looks like a merge/rebase gone wrong. why are those strings deleted? (same comment for the other files where they've been deleted)

Reply:
should be good now ✅